dtorrent/src/torrent_parser/parser.rs

```diff
@@ -1,6 +1,6 @@
 use std::{
     fs::File,
-    io::{BufReader, Error, Read},
+    io::{BufReader, Error, Read}, path::Path,
```

Comment:
If I'm not mistaken, a `cargo fmt` would move `path::Path` to the new line.
Running `cargo fmt` before marking a PR as ready is good practice since it makes the code style consistent between different developers.